src/hooks/editor/useToolbar.tsx

```
@@ -0,0 +1,148 @@
+    editor.on('update', handler);
+    editor.on('selectionUpdate', handler);
+
+    return () => {
+      editor.off('update', handler);
+      editor.off('selectionUpdate', handler);
+    };
+  }, [editor]);
+
+  useEffect(() => {
+    return () => {
+      blobUrlsRef.current.forEach((url) => URL.revokeObjectURL(url));
```

Biome (2.4.7)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)